src/GHC.hs

```diff
@@ -29,7 +29,7 @@ defaultKnownPackages =
     , filepath, genapply, genprimopcode, ghc, ghcBoot, ghcBootTh, ghcCabal, ghci
     , ghcPkg, ghcPrim, ghcTags, haddock, haskeline, hsc2hs, hoopl, hp2ps, hpc
     , hpcBin, integerGmp, integerSimple, iservBin, libffi, mkUserGuidePart
-    , parallel, pretty, primitive, process, rts, runGhc, stm, templateHaskell
+    , pretty, process, rts, runGhc, templateHaskell
```

Comment:
I don't think we should remove these packages from `defaultKnownPackages`, as it will then make it impossible to build them with Hadrian. I suggest you simply filter them out for now at the use site. The right way would probably be to use `getPackages` expression for extracting the list of packages that we actually build by default (see https://github.com/snowleopard/hadrian/blob/master/src/Rules.hs#L43 for example).